File: WebEmployeeApp/Models/SoccerGameDto.cs
using System;
using System.Collections.Generic;

namespace WebEmployeeApp.Models
{
    public class SoccerGameDto
    {
        public int Id { get; set; }
        public DateTime Created { get; set; }
        public string Name { get; set; }
        public string GameStatus { get; set; }
        public string ReporterId { get; set; }
        public List<SoccerEventDto> SoccerEvents { get; set; }
        public SoccerTeamDto HomeSoccerTeam { get; set; }
        public SoccerTeamDto GuestSoccerTeam { get; set; }
    }
}

File: WebEmployeeApp/ClientApp/src/app/models/SoccerGameDto.ts
import { SoccerEventDto } from "./SoccerEventDto";
import { SoccerTeamDto } from "./SoccerTeamDto";

export interface SoccerGameDto {
  id: string;
  created: string;
  name: string;
  gameStatus: string;
  reporterId: string;
  soccerEvents: SoccerEventDto[];
  homeSoccerTeam: SoccerTeamDto;
  guestSoccerTeam: SoccerTeamDto;
}
